include/crocoddyl/core/activations/quadratic-barrier.hpp

```diff
@@ -16,6 +16,8 @@
 #include "crocoddyl/core/utils/exception.hpp"
 #include "crocoddyl/core/activation-base.hpp"
 
+#include <pinocchio/utils/static-if.hpp>
```

Comment:
Is this works for CasaDi as well? 

are some other limitations?

Reply:
Yes, this is being used in casadi as well